### ISSUE TITLE
Alt attribute

### ISSUE THREAD

**Issue by jim-rosales** [Octavian1986/prj-rev-bwfs-dasmoto#2]
The [alt attribute](https://en.wikipedia.org/wiki/Alt_attribute) is used for both [accessibility](https://www.codecademy.com/articles/accessibility) and as a backup, in case your image cannot be rendered. This is why it is best practice to include the alt attribute in every image used on a site.

This [gist](https://gist.github.com/jim-rosales/6374f56d0c650ef829361da72bf1fc2b) shows an example of the alt attribute being used. 

https://github.com/Octavian1986/prj-rev-bwfs-dasmoto/blob/83e8090e27038132599113af0f1a95bed8c0fdf0/DasmotosArtsCrafts/index.html#L13
